src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1721,6 +1736,7 @@ protected override void VisitArguments(ImmutableArray<BoundExpression> arguments
             ImmutableArray<int> argsToParamsOpt,
             bool expanded)
         {
+            // PROTOTYPE(NullableReferenceTypes): What about conversions here?
```

Comment:
@cston I merged latest bits from the feature branch into this PR, to resolve conflicts. I think your PR on conversions missed some conversions here. Leaving a PROTOTYPE for now.